### DISCUSSION TITLE
scheduler card user experience in lovelace

### DISCUSSION THREAD

**Discussion by paobrunelli** [indomus/forum#3210] [HUB: Home Assistant]
Ho definito un'entità termostato ("controllo caldaia") realizzata con:
- Un termoigrometro Lumi che rileva la temperatura
- Un relè Shelly 1 che controlla il contatto pulito della caldaia
Ho quindi definito uno scheduler ("programma di riscaldamento settimanale") che controlla il suddetto termostato, in base a uno "schema" ("riscaldamento invernale") dello scheduler
Ho quindi semplicemente creato una dashboard che include due schede: termostato e scheduler, che si presenta così::
![image](https://github.com/user-attachments/assets/cda8f57c-00b8-4789-bbf5-759181f3ab84)
Cliccando sullo scheduler si apre lo schema del programma
![image](https://github.com/user-attachments/assets/ed836ba6-e526-40d7-8ea2-305b04da55f4)
Tutto funziona perfettamente, ma vorrei modificare l'esperienza utente della dashboard secondo le specifiche rappresentate nello schema sottostante
![image](https://github.com/user-attachments/assets/8904612f-62b4-4ec0-b539-bff2c4baa5d9)
Vale a dire: 
1. Lo schema attivo appare direttamente nello scheduler della scheda
2. Nel diagramma i colori dei blocchi orari sono diversi a seconda dello stato previsto (ad esempio rosso=off, verde=on) 
3. Una barra verticale rossa evidenzia l'ora corrente (risoluzione di 5 min.?)
4. La scheda termostato è attivamente collegata alla scheda scheduler, in modo che:
a. la modifica del setpoint di temperatura modifica quello del blocco orario corrente nello scheduler, 
b. una volta scaduto il blocco orario in cui il setpoint è stato sovrascritto, quello impostato nello schema prevale di nuovo
5. Sono disponibili due pulsanti:
a. Modifica: per modificare le impostazioni (ore e setpoint) dello schema attivo
b. Aggiungi: per aggiungere uno schema, renderlo attivo al posto di quello corrente

quanto è complesso configurare una scheda personalizzata del genere, almeno solo per il punto 1. e 2.?
grazie per qualsiasi supporto
Paolo

**Comment by indomus** [comment]
Ciao, se non converti il post in italiano siamo purtroppo costretti a riumuoverlo.
